### PR TITLE
chore: separate import statements by type

### PR DESCRIPTION
**Pull request by crodriguezvega**
## Description

Would be good to see if this can be automated when a file is saved...

Should we add this convention to a coding guidelines document?

closes: #382 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by colin-axner — APPROVED** [review]

**Comment by colin-axner** [comment]
> Should we add this convention to a coding guidelines document?

Yes!

File: modules/core/02-client/client/cli/tx.go [hunk 1]
```diff
@@ -17,6 +14,9 @@ import (
 	govcli "github.com/cosmos/cosmos-sdk/x/gov/client/cli"
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	"github.com/pkg/errors"
```
Comment (damiannolan) [inline modules/core/02-client/client/cli/tx.go:17]: Random question: Any ideas why this is used over std lib `errors`? I think that repo is archived. cc @colin-axner

**Review by damiannolan — APPROVED** [review]

**Review by seantking — APPROVED** [review]

File: modules/core/02-client/client/cli/tx.go (hunk 1 above)
Comment (colin-axner) [reply modules/core/02-client/client/cli/tx.go:17]: No idea. Want to open an issue?

Comment (damiannolan) [reply modules/core/02-client/client/cli/tx.go:17]: Sure. I can do! :)

**Pull request merged**